src/group_theory/perm/sign.lean

```diff
@@ -139,13 +141,13 @@ variable [decidable_eq α]
 def support [fintype α] (f : perm α) := univ.filter (λ x, f x ≠ x)
 
 @[simp] lemma mem_support [fintype α] {f : perm α} {x : α} : x ∈ f.support ↔ f x ≠ x :=
-by simp [support]
+by simp only [support, true_and, mem_filter, mem_univ]
 
 def is_swap (f : perm α) := ∃ x y, x ≠ y ∧ f = swap x y
 
 lemma swap_mul_eq_mul_swap (f : perm α) (x y : α) : swap x y * f = f * swap (f⁻¹ x) (f⁻¹ y) :=
 equiv.ext $ λ z, begin
-  simp [mul_apply, swap_apply_def],
+  simp only [perm.mul_apply, swap_apply_def],
   split_ifs;
   simp [*, eq_inv_iff_eq] at * <|> cc
```

Comment:
How many goals is this closing? I guess you could `squeeze_simp` all of the goals, and merge the resulting simp-sets.

Reply:
ooh, the `<|> cc` isn't necessary. Maybe some simp lemmas have been added since this file was written.